src/main/kotlin/com/terraformation/backend/accelerator/api/DeliverablesController.kt

```diff
@@ -0,0 +1,207 @@
+        DeliverablePayload(
+            category = DeliverableCategory.LegalEligibility,
+            descriptionHtml = "<p>A description</p>",
+            documents =
+                listOf(
+                    SubmissionDocumentPayload(
+                        Instant.now(),
+                        "Project's articles of incorporation",
+                        DocumentStore.Google,
+                        SubmissionDocumentId(13974),
+                        "Incorporation Documents_2024-02-28_Omega_Projects articles of incorporation.doc",
+                    )),
+            feedback = "Is this a joke? This is a bunch of cat photos, not a legal document.",
+            id = deliverableId,
+            internalComment = "These guys are real jokers.",
+            name = "Incorporation Documents",
+            organizationId = OrganizationId(1),
+            organizationName = "Test Org",
+            participantId = ParticipantId(2),
+            participantName = "Random Participant",
+            projectId = ProjectId(3),
+            projectName = "Omega Project",
+            status = SubmissionStatus.Rejected,
+            type = DeliverableType.Document,
+        ))
+  }
+
+  @ApiResponse(
+      responseCode = "307",
+      description =
+          "If the current user has permission to view the document, redirects to the document " +
+              "on the document store. Depending on the document store, the redirect URL may or " +
+              "may not be valid for only a limited time.")
+  @ApiResponse404
+  @GetMapping("/{deliverableId}/documents/{documentId}")
+  @Operation(summary = "Gets a single submission document from a deliverable.")
+  fun getDeliverableDocument(
+      @PathVariable deliverableId: DeliverableId,
+      @PathVariable documentId: SubmissionDocumentId
+  ): RedirectView {
+    val url = URI("https://dropbox.com/")
+
+    val redirectView = RedirectView(url.toString(), false, false)
+    redirectView.setStatusCode(HttpStatus.TEMPORARY_REDIRECT)
+
+    return redirectView
+  }
+}
+
+data class ListDeliverablesElement(
+    val category: DeliverableCategory,
+    @Schema(description = "Optional description of the deliverable in HTML form.")
+    val descriptionHtml: String?,
+    val id: DeliverableId,
+    val name: String,
+    @Schema(
+        description =
+            "Number of documents submitted for this deliverable. Only valid for deliverables of " +
+                "type Document.")
+    val numDocuments: Int?,
+    val organizationId: OrganizationId,
+    val organizationName: String,
+    val participantId: ParticipantId,
+    val participantName: String,
+    val projectId: ProjectId,
+    val projectName: String,
+    val status: SubmissionStatus,
+    val type: DeliverableType,
+)
+
+data class SubmissionDocumentPayload(
+    val createdTime: Instant,
+    val description: String,
+    val documentStore: DocumentStore,
+    val id: SubmissionDocumentId,
+    val name: String,
+)
```

Comment:
This doesn't include a download URL because, at least on Dropbox, it requires an API call to generate each URL which would add latency and another possible point of failure. The link should point to the document download endpoint (defined in the controller class above) which the client can generate.